lib/adapters/log.js

```diff
@@ -2,7 +2,7 @@
 
 var _ = require('lodash')
 
-module.exports = function log (options, context, auth, routes, done) {
+module.exports = function log (options, context, auth, middleware, routes, done) {
```

Comment:
Don't change the api contract between `seneca-web` and adapters.  Remove `middleware` parameter here

Reply:
noted